cea/demand/metamodel/nn_generator/nn_test_sampler.py

```diff
@@ -67,17 +69,15 @@ def sampling_single(locator, random_variables, target_parameters, list_building_
     return urban_input_matrix, urban_taget_matrix
 
 
-def run_as_script():
+def main(config):
     gv = cea.globalvar.GlobalVariables()
-    scenario_path = gv.scenario_reference
-    locator = cea.inputlocator.InputLocator(scenario_path=scenario_path)
-    weather_path = locator.get_default_weather()
+    locator = cea.inputlocator.InputLocator(scenario_path=config.scenario)
```

Comment:
I think the parameter to `InputLocator` is now called `scenario`

Reply:
fixed